examples/Commands/TimeSeries/ts_info.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Predis\Client;
+use Predis\Command\Argument\TimeSeries\CommonArguments;
+use Predis\Command\Argument\TimeSeries\CreateArguments;
+
+require __DIR__ . '/../../shared.php';
+
+// Example of TS.CREATE command usage:
+
+// 1. Create time series
+$client = new Client();
+
+$arguments = (new CreateArguments())
+    ->retention(60000)
```

Comment:
IMHO we called it in python retention_msecs as a variable. The reason - making it clear that its in milliseconds. WDYT?

Reply:
@chayim Sure, let's make it more consistent